design/one-pager-stack-relationship-labels.md

```diff
@@ -0,0 +1,165 @@
+# Stack Parent/Child Relationship Label Spec
+
+- Owner: Steven Rathbauer (@rathpc)
+- Reviewers: Crossplane Maintainers
+- Status: Draft
+
+## Proposal
+
+[crossplaneio/crossplane#934](https://github.com/crossplaneio/crossplane/issues/934)
+
+## Problem
+
+There is currently no easy way to get a list of all resources that are owned by a parent resource.
+We can leverage CRD labels to make this easier. As long as we have a standard format for doing so.
+
+Per the proposal, it seems that the closest example of what I am referring to is shown on this page
+https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/#labels within the
+`app.kubernetes.io/part-of` label. However since we are not exactly using these labels for the
+intentions defined within that page, we should use something following the patterns instead.
+
+As long as we define this label on all children CRD's, we should then be able to query all children
+of a given parent name using those labels.
```

Comment:
Nitpick: `parent name` isn't entirely accurate, since we need the kind as well. I would probably use `parent object` instead.